src/java.base/share/classes/java/nio/channels/Channels.java

```diff
@@ -423,6 +423,9 @@ protected void implCloseChannel() throws IOException {
      * the {@link Reader#mark mark} or {@link Reader#reset reset} methods.
      * Closing the stream will in turn cause the channel to be closed.  </p>
      *
+     * @implNote
+     * The implementation may ignore the value of {@code minBufferCap} provided.
```

Comment:
I think we'll need to adjust the specification of parameter too, something like "may be ignored".

For the implNote I think it can just sat that minBufferCap is ignored.

Reply:
Adjusted in 5f93e5776474cc286ea32963fb55bf05cf6ce22f.